frontend/src/components/course/components/edit-category-modal/edit-category-modal.tsx

```diff
@@ -49,6 +54,14 @@ const EditCategoryModal: FC<Props> = ({
     dispatch(courseActions.updateCategory({ courseId, newCategoryId }))
       .unwrap()
       .then(() => {
+        if (isCourseCategoriesPage) {
+          dispatch(
+            courseCategoriesActions.getCourses({
+              page: 1,
+              count: PaginationDefaultValue.DEFAULT_COURSE_CATEGORIES_COUNT,
+            }),
+          );
+        }
```

Comment:
I see this approach first time, seems cool. I have a question if the functionality here is able to be moved to actions? If it is right to leave functionality as it is or move it to actions?

Reply:
@WilhelmWesser I thought about that, and decided to use it here, just where I'm using useLocation hook - didn't see the using hooks in actions previously